.github/workflows/upload.yml

```diff
@@ -12,20 +12,18 @@ jobs:
       - name: Checkout Git
         uses: actions/checkout@v2
 
-      - name: Install Python 3
-        uses: actions/setup-python@v2
+      - name: Install poetry
+        uses: abatilo/actions-poetry@v2
         with:
-          python-version: 3.9
+          poetry-version: "1.7.1"
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install pipenv
-          pipenv install --dev
+          poetry install
 
       - name: Build package
         run: |
-          pipenv run python -m build --no-isolation
+          make build
 
       - name: Publish package
```

Comment:
I expected some changes here as well since you updated the makefile to support publishing via poetry? Is there any known limitations/reason for skipping it?

Reply:
I'm not 100% sure on if `gh-action-pypi-publish` will work but we're building to `build` still so theoretically it should